test/e2e/node/pods.go

```diff
@@ -220,5 +205,60 @@ var _ = SIGDescribe("Pods Extended", func() {
 			Expect(err).NotTo(HaveOccurred(), "failed to query for pod")
 			Expect(pod.Status.QOSClass == v1.PodQOSGuaranteed)
 		})
+
+		It("should be set on Pods with matching resource requests and limits for memory", func() {
```

Comment:
I would prefer to see one test case for `PodQOSBurstable` and another for `PodQOSBestEffort`; it's unclear to me what the benefit is of having them both in a single test case